frontend/src/scenes/apps/appMetricsSceneLogic.ts

```diff
@@ -217,31 +223,44 @@ export const appMetricsSceneLogic = kea<appMetricsSceneLogicType>([
     })),
 
     actionToUrl(({ values, props }) => ({
-        setActiveTab: () => {
-            if (values.activeTab === AppMetricsTab.HistoricalExports) {
-                return urls.appHistoricalExports(props.pluginConfigId)
-            }
-
-            return urls.appMetrics(props.pluginConfigId, values.activeTab ?? undefined)
-        },
+        setActiveTab: () => getUrl(values, props),
+        setDateFrom: () => getUrl(values, props),
     })),
 
     urlToAction(({ values, actions, props }) => ({
-        '/app/:pluginConfigId/:page': (
-            url: Record<string, string | undefined>,
-            params: Record<string, string | undefined>
-        ) => {
+        '/app/:pluginConfigId/:page': (url: Record<string, string | undefined>, params: AppMetricsUrlParams) => {
             // :KLUDGE: Only handle actions if this logic is active
             if (props.pluginConfigId === Number(url.pluginConfigId)) {
                 if (!values.pluginConfig) {
                     actions.loadPluginConfig()
                 }
                 if (url.page === AppMetricsTab.HistoricalExports) {
                     actions.setActiveTab(AppMetricsTab.HistoricalExports)
-                } else if (params.tab && INITIAL_TABS.includes(params.tab as any)) {
-                    actions.setActiveTab(params.tab as AppMetricsTab)
+                } else {
+                    if (params.tab && INITIAL_TABS.includes(params.tab as any)) {
+                        actions.setActiveTab(params.tab as AppMetricsTab)
+                    }
+                    if (params.from) {
+                        actions.setDateFrom(params.from)
+                    }
                 }
             }
         },
     })),
 ])
+
+function getUrl(values: appMetricsSceneLogicType['values'], props: appMetricsSceneLogicType['props']): string {
+    if (values.activeTab === AppMetricsTab.HistoricalExports) {
+        return urls.appHistoricalExports(props.pluginConfigId)
+    }
+
+    const params = {}
+    if (values.activeTab) {
+        params['tab'] = values.activeTab
+    }
+    if (values.dateFrom !== DEFAULT_DATE_FROM) {
```

Comment:
I'd still keep that in the url, more convenient if someone is messing around with the url directly, plus we might change the default in the future so their historical bookmarked links are still the same